apps/about/stylesheets/home.styl

```diff
@@ -0,0 +1,207 @@
+        width auto
+        right auto
+        overflow hidden
+        max-width 760px
+        img
+          display block
+          margin 0 auto
+      +mobile()
+        &__copy h1
+          font-size 25px
+          line-height 1.3
+
+
+  &--features
+    max-width 900px
+    margin (unit * 3) auto (unit * 6)
+    font-size 16px
+    +below(1100px)
+      height auto
+      padding 0 unit
+    &__introduction
+      p()
+      font-size 1.2rem
+      max-width 600px
+      margin unit auto (unit * 4)
+      +mobile()
+        width 90%
+    h2, h1
+      text-align center
+      font-weight normal
+      color black
+      margin-top 0
+    h1
+      max-width 600px
+      margin 0 auto
+      +mobile()
+        width 90%
+    &__cta
+      text-align center
+    &__list
+      padding (unit * 1.5)
+      &__feature
+        width 280px
+        display inline-block
+        text-align center
+        color black
+        vertical-align top
+        padding unit * 1.5
+        +below(900px)
+          display block
+          margin unit auto
+        &__image
+          margin-bottom unit
+          img
+            width 170px
+            height 130px
+            border 1px solid bordergray
+            &:hover
+              border-color textgray !important
+        &__title
+          margin unit 0
+        &__desc
+          p()
+          font-size 14px
+          font-size size
+          color textgray
+
+  &--diatribe
+    max-width 600px
+    margin (unit * 4) auto (unit * 6)
+    padding 0 unit
+    h2, h1
+      text-align center
+      font-weight normal
+      color black
+      margin-top 0
+    &__cta
+      margin (unit * 3) 0 0
+    p
+      p()
+      font-size 1.2rem
+      margin unit auto
+    img
+      margin unit 0
+      width 60%
+    +mobile()
+      p
+        width 90%
+        font-size 1.1rem
+  &--posts
+    &__headline
+      text-align center
+      font-weight normal
+      color black
+      margin-top 0
+      margin-bottom unit * 2
+    &__post
+      background-position center center
+      background-size cover
+      height 300px
+      line-height 60px
+      display block
+      input.btn
+        margin-top unit
+        color black
+        border-color black
+        font-size 16px
+        margin-top 30px
+        width 150px
+        line-height 60px
+        &:hover
+          background-color black
+          color white
+      h1
+        font-size 35px
+        font-weight normal
+        line-height 40px
+        margin 0
+      &__overlay
+        background-color rgba(255,255,255,0.95)
+        width 100%
+        height 100%
+        &:hover
+          background-color rgba(255,255,255,0.8)
+        &__content
+          position relative
+          top 50%
+          transform translateY(-50%)
+          color black
+          text-align center
+
+      +below(900px)
+        height 390px
+        h1
+          font-size 30px
+          line-height 40px
```

Comment:
This is just copy/pasted the old homepage style. I'm not going to untangle what's relevant.  Realistically we should do a consistency pass on the about/info pages because they look rather out of place now.